web/docs/use/run/README.md

```diff
@@ -22,48 +27,48 @@ multiple clients that publish events and submit queries:
 
 ## Start a server
 
-The `start` command spins up a VAST server that blocks until told to
+The `start` command spins up a Tenzir server that blocks until told to
 [stop](#stop-a-server):
 
 ```bash
-vast start
+tenzirctl start
```

Comment:
This should mention tenzird instead?

Reply:
Yes, but the whole document will need to be rewritten. I noted it down.